### PR TITLE
Add the #bodhi channel to the README.

### PR DESCRIPTION
**Pull request by bowlofeggs**

### CODE REVIEW

**Comment by trishnaguha** [comment]
:100: :+1:

File: README.rst [hunk 1]
```diff
@@ -207,3 +207,9 @@ There is a meeting every four weeks between Bodhi developers and stakeholder,
 held on IRC. If you would like to attend, you can see details here:
 
 https://apps.fedoraproject.org/calendar/meeting/4667/
+
+
+IRC
+===
+
+Come join us in the #bodhi channel on `Freenode <https://freenode.net/>`_.
```
Comment (pypingou) [inline README.rst]: #bodhi only or #fedora-apps as well?

Comment (bowlofeggs) [reply README.rst]: @pypingou I could alter it to be both. I could also just only put #fedora-apps. The only reason I made the #bodhi channel was that I recently learned that we have at least two users that are not Fedora (which is exciting!)

What is your preference?

Comment (pypingou) [reply README.rst]: I would put both, the more the merrier :)

**Comment by ankit01ojha** [comment]
Is the issue closed or open?

**Comment by trishnaguha** [comment]
@ankit01ojha this is a PR.

**Comment by ankit01ojha** [comment]
Thanks, I thought it was open.

**Comment by pypingou** [comment]
Well it is, since it hasn't been merged :)

**Comment by bowlofeggs** [comment]
@pypingou I've altered this PR to mention both channels. How is this?

**Comment by pypingou** [comment]
:+1: for me, thanks :)

**Pull request merged**